newdle/client/src/Calendar.js

```diff
@@ -0,0 +1,46 @@
+import 'react-dates/initialize';
+import React, {useCallback, useEffect} from 'react';
+import {HTML5_FMT} from 'moment';
+import {useDispatch, useSelector} from 'react-redux';
+import {DayPickerSingleDateController as DayPicker} from 'react-dates';
+import {getCalendarDates, getCalendarActiveDate} from './selectors';
+import {setActiveDate, clearCalendarActiveDate} from './actions';
+import {serializeDate, toMoment} from './util/date';
+
+import 'react-dates/lib/css/_datepicker.css';
+import styles from './Calendar.module.scss';
+
+export default function Calendar() {
+  const calendarDates = useSelector(getCalendarDates);
+  const activeDate = useSelector(getCalendarActiveDate);
+  const dispatch = useDispatch();
+
+  const onDateChange = useCallback(date => dispatch(setActiveDate(serializeDate(date))), [
```

Comment:
If this data is only used inside this component, does it really belong in the redux store?

I'd only put things in the store that have to persist between unmounts of whatever component deals with the data. So for example, the participants, time slots and duration, since switching between steps shouldn't lose that data.

Reply:
I assumed this will be somehow shared between the `Timeline` component and `Calendar` hence I put it in the store. I might be missing some details since I wasn't there during the design stage.

Reply:
And apart from that, I think that selecting a date will have impact on the `Timeline` component.

Reply:
Yes you are right, the timeline needs to know the selected date.